app/services/identity/transfer.rb

```diff
@@ -24,6 +24,9 @@ def initialize(from_user:, to_user:)
 
     def transfer_identities!
       teacher_profile = TeacherProfile.find_or_create_by!(user: to_user)
+      unless teacher_profile.trn
```

Comment:
@tonyheadford @slawosz I think this is worth chatting about, to confirm teacher profiles are only populated with a trn if the trn is validated? We only dedupe users when there are two teacher profiles with the same trn example https://github.com/DFE-Digital/early-careers-framework/blob/main/app/services/npq/dedupe_participant.rb#L31 so we should never get to this point where we are merging users without a trn and need this?